besom-cfg/k8s/project.scala

```diff
@@ -0,0 +1,23 @@
+//> using scala 3.3.3
+
+//> using dep com.lihaoyi::os-lib::0.9.3
+//> using dep org.virtuslab::besom-cfg:0.2.0-SNAPSHOT
```

Comment:
this will be replaced by current besom version next time you run the version bump script, that's why I was suggesting the version alignment ;)

Reply:
so there has to be a way to provide exceptions in the version bump script